src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java

```diff
@@ -74,8 +74,14 @@ private boolean isLineMatching(AuditEvent event) {
         boolean result = false;
         if (event.getLocalizedMessage() != null) {
             for (List<Integer> aLineRangeList : lineRangeList) {
-                result = aLineRangeList.get(0) < event.getLine()
-                        && event.getLine() < aLineRangeList.get(1);
+                final int startLine = aLineRangeList.get(0);
+                final int endLine = aLineRangeList.get(1);
+                if (startLine == endLine) {
+                    result = startLine == event.getLine();
+                }
+                else {
+                    result = startLine < event.getLine() && event.getLine() <= endLine;
+                }
```

Comment:
```
int currentLine = event.getLine();
result = currentLine >= startLine && currentLine <= endLine;
```

Reply:
done

Reply:
It is not done. The whole `if-else` block had to be replaced with those 2 lines.

Reply:
moved to https://github.com/checkstyle/patch-filters/issues/38, should be done asap